### DISCUSSION TITLE
PLots are invalid !!

### DISCUSSION THREAD

**Discussion by bashar1988** [swar/Swar-Chia-Plot-Manager#310] [General]
iam trying to make plots with SWAR since 2 days but the new plots are alyways invalid, can any one help me please and tell me where is the problem 
![image](https://user-images.githubusercontent.com/83901628/118176055-3295b000-b431-11eb-9523-a6ea85f09b2c.png)

**Comment by raid5** [comment]
I think this is probably more suited for the Chia team, as the Swar Manager just use the `chia` executable under the hood, similar to the Chia GUI. But maybe someone has his this chia error before and can explain why it's happening. I've seen this src size error on plots create with the CLI and GUI as well.

**Reply by gcacademy** [reply]
seem to me, I use this make plots,but all plots from swar are invalid. please tell me how to do or how to fix?THANKS.

**Comment by gcacademy** [comment]
how do you fix it?
